cmd/egnkey/generate.go

```diff
@@ -153,6 +220,19 @@ func generateBlsKeys(numKeys int, path string, passwordFile, privateKeyFile *os.
 func generateECDSAKeys(numKeys int, path string, passwordFile, privateKeyFile *os.File) error {
 	for i := 0; i < numKeys; i++ {
 		key, err := crypto.GenerateKey()
+		privateKeyHex := hex.EncodeToString(key.D.Bytes())
+
+		// Low tech fix for creating valid keys, probably better is validation code
+		// Checking if the length of privateKeyHex is 32 bytes (64 characters)
+		for {
+			if len(privateKeyHex) != 64 {
+				key, err = crypto.GenerateKey()
+				privateKeyHex = hex.EncodeToString(key.D.Bytes())
+			} else {
+				break
```

Comment:
I don't think we need a for loop to check this - we should just ignore the key and let the upper loop continue - I actually already have a PR for this - https://github.com/Layr-Labs/eigensdk-go/pull/60